src/hapi/pipelines/app/__main__.py

```diff
@@ -64,6 +64,58 @@ def parse_args():
     return parser.parse_args()
 
 
+def combine_default(country, default):
+    country["input"] = country["input"] + default["input"]
+    country["output"] = country["output"] + default["output"]
+    country["output_hxl"] = country["output_hxl"] + default["output_hxl"]
+    return country
+
+
+def find_defaults(config):
+    default_list = []
+    for key in config:
+        key_components = key.split("_")
+        if len(key_components) == 2 and key_components[1] == "default":
```

Comment:
The length may not always be 2, e.g. "operational_presence_defaults". Maybe check that the length is > 1 instead?